dss/stepfunctions/visitation/__init__.py

```diff
@@ -0,0 +1,161 @@
+
+import copy
+import typing
+from uuid import uuid4
+from enum import Enum, auto
+from .. import step_functions_invoke
+
+
+class DSSVisitationException(Exception):
+    pass
+
+
+class DSSVisitationExceptionRetry(DSSVisitationException):
+    pass
+
+
+class AttrGetter:
```

Comment:
This seems superfluous. Is this to silence mypy?

Reply:
Let's please remove this. You can suppress mypy warnings on any given line with ` . # type: ignore`

Reply:
Removed.